pacman/model/graphs/abstract_graph.py

```diff
@@ -220,12 +220,3 @@ def get_outgoing_edge_partition_starting_at_vertex(
         :rtype:\
             :py:class:`pacman.model.graphs.abstract_outgoing_edge_partition.AbstractOutgoingEdgePartition`
         """
-
-    @abstractmethod
-    def get_outgoing_edge_partitions_with_traffic_type(self, traffic_type):
-        """ Get the outgoing edge partitions with a given traffic type
-
```

Comment:
I'm curious as to why this is being removed as part of this changeset.

Reply:
nnone uses it at all. dead code. so clean up